krabs/krabs/ut.hpp

```diff
@@ -19,6 +19,14 @@ namespace krabs { namespace details {
     struct ut {
 
         typedef krabs::provider<> provider_type;
+		
+		struct FilterSettings
+		{
```

Comment:
Use snake_case: `filter_settings`. Also, the opening brace should be on the previous line:

```suggestion
		struct filter_settings {
```

Reply:
Done